ui/imports/shared/popups/SendModal.qml

```diff
@@ -95,6 +95,12 @@ StatusDialog {
             if(errorType === Constants.SendAmountExceedsBalance)
                 bestRoutes = []
         }
+
+        function setSelectedAccount() {
+            popup.selectedAccount = header.model.get(header.selectedIndex)
```

Comment:
@Khushboo-dev-cpp there are some things I conceptually do not understand.

By setting `selectedAccount` here, you're breaking a binding made in the line `39     property var currentAccount: walletSectionCurrent` is that what we want? Is selectedAccount fully maintained by the backed or you're using it only on the qml side?

Reply:
@alaibe see this, the model is pointing to send accounts itself via the store